### PR TITLE
doc: document that users can self-assign issues

### PR DESCRIPTION
**Pull request by KentHsu**
# Description

Refer to [js-sdk](https://github.com/dapr/js-sdk/pull/527), informs users about the GitHub daprbot commands available in the repo with doc link.

## Issue reference

Please reference the issue this PR will close: #602

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

**Comment by elena-kolevska** [comment]
Thanks Kent!
I think we should move the new paragraph at the end of the doc, though, because it doesn't read very well in continuation with the sentence before it ("When contributing to the [Python SDK](https://github.com/dapr/python-sdk) the following rules and best-practices should be followed").

I also think we should add the same paragraph in the `README.md` file, where most developers would probably look first. You can add it after "Generate gRPC Protobuf client" and before "Help & Feedback".

File: daprdocs/content/en/python-sdk-contributing/python-contributing.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ description: Guidelines for contributing to the Dapr Python SDK
 
 When contributing to the [Python SDK](https://github.com/dapr/python-sdk) the following rules and best-practices should be followed.
 
+## Github Dapr Bot Commands
```
Comment (berndverst) [inline daprdocs/content/en/python-sdk-contributing/python-contributing.md:11]: Move to the end of the docs.

**Review by berndverst — CHANGES_REQUESTED** [review]

**Comment by KentHsu** [comment]
I movedthe new paragraph to the end of python-contributing.md and decide to add the same paragraph in CONTRIBUTING.md. Please let me know if there are other documentations you want to update.

File: daprdocs/content/en/python-sdk-contributing/python-contributing.md (hunk 1 above)
Comment (KentHsu) [reply daprdocs/content/en/python-sdk-contributing/python-contributing.md]: Updated

**Review by elena-kolevska — APPROVED** [review]
Looks great! Thanks @KentHsu!

**Review by berndverst — APPROVED** [review]

**Pull request merged**